src/symbol/relocate.rs

```diff
@@ -199,4 +199,31 @@ impl RelocatedDwarf {
         }
         Ok(None)
     }
+
+    /// Find a address for each substatement in the `Location`.
+    /// This function return a hashmap where the key are the column number of the statement,
+    /// and where the value is a address of a instruction from that statement.
+    /// If the `Location` specify a column number, the hashmap will only contain one value.
```

Comment:
Nitpicking some typos (similar thing about the other docs for `find_location_addr`):
```suggestion
    /// Find an address for each substatement in the `Location`.
    /// This function returns a hashmap where a key is a column number of each statement
    /// and a value is an address of an instruction from that statement.
    /// If the `Location` specifies a column number, the hashmap will only contain one value.
```